ui/stops/OBAStopViewController.m

```diff
@@ -69,6 +85,7 @@ - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
     
     self.refreshTimer = [NSTimer scheduledTimerWithTimeInterval:kRefreshTimeInterval target:self selector:@selector(reloadData:) userInfo:nil repeats:YES];
+    self.apptentiveTimer = [NSTimer scheduledTimerWithTimeInterval:10 target:self selector:@selector(recordUserVisit:) userInfo:nil repeats:NO];
```

Comment:
Seems like this would be cleaner with a `performSelector...afterDelay` or a `dispatch_after` instead of a one-shot timer.